dist/designer/xoom

```diff
@@ -1,2 +1,3 @@
 #!/bin/bash
-java -jar $VLINGO_XOOM_DESIGNER_HOME/bin/xoom-designer-dist.jar $@ --currentDirectory ${PWD}
+export VLINGO_XOOM_DESIGNER_HOME="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+java -jar $VLINGO_XOOM_DESIGNER_HOME/bin/vlingo-xoom-starter-dist.jar $@ --currentDirectory ${PWD}
```

Comment:
Sorry, didn't notice it at first. Jar name is `xoom-designer-dist.jar` now.

Reply:
done